custom/css.json

```diff
@@ -887,8 +887,7 @@
     },
     "text-underline-position": {
       "__additional_values": {
-        "above_below": ["above", "below"],
-        "left_right": ["left", "right"]
+        "above_below": ["above", "below"]
```

Comment:
We should also remove `above_below` and add the individual entries following what we discussed in the BCD PR as well.